tools/schema_loader.cc

```diff
@@ -545,6 +546,38 @@ schema_ptr load_schema_from_statistics(sstring keyspace, sstring table, const ss
     return builder.build();
 }
 
+sstring disk_string_to_string(const sstables::disk_string<uint32_t>& ds) {
+    return sstring(ds.value.begin(), ds.value.end());
+}
+
+column_kind from_sstable_column_kind(sstables::sstable_column_kind k) {
+    switch (k) {
+        case sstables::sstable_column_kind::partition_key: return column_kind::partition_key;
+        case sstables::sstable_column_kind::clustering_key: return column_kind::clustering_key;
+        case sstables::sstable_column_kind::static_column: return column_kind::static_column;
+        case sstables::sstable_column_kind::regular_column: return column_kind::regular_column;
+    }
```

Comment:
Need on_internal_error() here for gcc. Not sure why clang doesn't complain.

Reply:
Added in v3.